projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -90,13 +98,37 @@ export class TuiInputPhoneInternational extends TuiControl<string> {
     @ViewChild(MaskitoDirective, {read: ElementRef})
     private readonly input?: ElementRef<HTMLInputElement>;
 
+    @ViewChildren('button', {read: ElementRef})
```

Comment:
We can drop `#button` from template and use `TuiOption` here instead. And let's call the array `options`.